client-network/src/network_ops/default_network_ops_client.rs

```diff
@@ -472,8 +445,52 @@ where
     }
 
     #[inline]
-    fn get_staked_state(&self, address: &StakedStateAddress) -> Result<StakedState> {
-        self.get_staked_state_account(address)
+    fn get_staked_state(
+        &self,
+        name: &str,
+        address: &StakedStateAddress,
+        verify: bool,
+    ) -> Result<StakedState> {
+        let mstaking = if verify {
+            let sync_state = self.wallet_client.get_sync_state(name)?;
+            let rsp = self.client.query(
```

Comment:
full name with "response"

Reply:
Personally, I think well-known shortcuts should be ok to keep the code short, actually easier to read.